oggm/cfg.py

```diff
@@ -182,6 +182,14 @@ def doc_str(self, key):
 _doc = ''
 BASENAMES['past_model'] = ('past_model.nc', _doc)
 
+_doc = 'Calving flux values'
+BASENAMES['calving_fluxes'] = ('calving_fluxes.pkl', _doc)
+
+_doc = 'Water depth'
+BASENAMES['water_depth'] = ('water_depth.pkl', _doc)
+
+_doc = 'Thickness at the calving front'
+BASENAMES['H_ice'] = ('H_ice.pkl', _doc)
 
```

Comment:
In order to avoid to have too many entries in ``BASENAMES``, I suggest to define only one entry here:

```python
+_doc = 'Dict of calving flux values, water depth, and thickness at the calving front'
+BASENAMES['calving'] = ('calving.pkl', _doc)
```
Unless there is a good reason to have three entries (e.g. the data is written and read from two different tasks).

See my comment below for how this translates in your code.